lib/sm/api/attachments.rb

```diff
@@ -1,14 +0,0 @@
-# frozen_string_literal: true
-require 'common/models/collection'
-
-module SM
-  module API
-    module Attachments
```

Comment:
moved this inside of messages, since it is just one method and technically it is a nested resource.